NDB.Covid19/NDB.Covid19.iOS/Views/AuthenticationFlow/QuestionnaireCountries/QuestionnaireCountriesViewController.cs

```diff
@@ -51,12 +51,20 @@ public override void ViewWillDisappear(bool animated)
 
         void SetStyling()
         {
-            StyleUtil.InitLabelWithSpacing(TitleLbl, StyleUtil.FontType.FontBold, QuestionnaireCountriesViewModel.COUNTRY_QUESTIONAIRE_HEADER_TEXT, 1.14, 24, 38);
-            StyleUtil.InitLabelWithSpacing(SubtitleLbl, StyleUtil.FontType.FontRegular, QuestionnaireCountriesViewModel.COUNTRY_QUESTIONAIRE_INFORMATION_TEXT, 1.28, 16, 20);
+            StyleUtil.InitLabelWithSpacing(TitleLbl, StyleUtil.FontType.FontBold, QuestionnaireCountriesViewModel.COUNTRY_QUESTIONAIRE_HEADER_TEXT, 1.14, 32, 38);
+            StyleUtil.InitLabelWithSpacing(SubtitleLbl, StyleUtil.FontType.FontRegular, QuestionnaireCountriesViewModel.COUNTRY_QUESTIONAIRE_INFORMATION_TEXT, 1.28, 20, 22);
             StyleUtil.InitLabelWithSpacing(ListExplainLbl, StyleUtil.FontType.FontRegular, QuestionnaireCountriesViewModel.COUNTRY_QUESTIONAIRE_FOOTER, 1.28, 16, 20);
-            NextBtn.SetTitle(QuestionnaireCountriesViewModel.COUNTRY_QUESTIONAIRE_BUTTON_TEXT, UIControlState.Normal);
-            ButtonView.Alpha = 0.9f;
-            ButtonView.BackgroundColor = "#001F34".ToUIColor();
+            StyleUtil.InitButtonStyling(NextBtn, QuestionnaireCountriesViewModel.COUNTRY_QUESTIONAIRE_BUTTON_TEXT);
+            SetButtonViewStyle();
+        }
+
+        void SetButtonViewStyle()
+        {
+            ButtonView.BackgroundColor = ColorHelper.DEFAULT_BACKGROUND_COLOR;
+            ButtonView.Layer.MasksToBounds = false;
+            ButtonView.Layer.ShadowOffset = new CoreGraphics.CGSize(0, 5);
+            ButtonView.Layer.ShadowRadius = 5;
```

Comment:
Don't we have an already specified size of this radius in a shared file for all iOS files?

Reply:
No, this is for the shadow on the view containing the button, and not the button itself.